led-raspberrypi/vendor/modules.txt

```diff
@@ -1,50 +0,0 @@
-# github.com/eclipse/paho.mqtt.golang v1.2.0
```

Comment:
im thinking that maybe we should remove all the vendor and use `gomod` mode. do you agree?